homeassistant/components/glinet/translations/en.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Make sure to remove this file. It is automatically generated by Localize based on `strings.json`.

Reply:
removed